kubejs/server_scripts/enigmatica/kubejs/base/recipes/replace_input.js

```diff
@@ -27,6 +27,14 @@ events.listen('recipes', (event) => {
     event.replaceInput({}, 'pedestals:dustflour', '#forge:dusts/flour');
     event.replaceInput({}, 'create:dough', '#forge:doughs');
     event.replaceInput({}, 'farmersdelight:wheat_dough', '#forge:doughs');
+    event.replaceInput({}, 'create:bar_of_chocolate', '#forge:chocolate_bars');
+    event.replaceInput({}, 'simplefarming:chocolate', '#forge:chocolate_bars');
+    event.replaceInput({}, 'simplefarming:noodles', '#forge:pasta/raw_pasta');
+    event.replaceInput({}, 'simplefarming:jam', '#forge:jams');
+    event.replaceInput({}, 'simplefarming:raw_bacon', '#forge:raw_bacon');
+    event.replaceInput({}, 'simplefarming:cooked_bacon', '#forge:cooked_bacon');
+    event.replaceInput({ mod: 'simplefarming' }, 'minecraft:cooked_chicken', '#forge:raw_chicken');

```

Comment:
Are you trying to give people salmonella? :P

Reply:
I took the liberty of changing `forge:raw_chicken` to `forge:cooked_chicken`, I hope that's what you intended :)

Reply:
Oops, not sure how I messed that up 😅